src/server/project.ts

```diff
@@ -142,6 +154,35 @@ namespace ts.server {
             return this.cachedUnresolvedImportsPerFile;
         }
 
+        public static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void): {} {

```

Comment:
And I'm sure someone like @basart would like to support angular and other plugins in his cloud IDE.